app/services/arcgis_api/geocoder.rb

```diff
@@ -112,7 +128,7 @@ def parse_suggestions(response_body)
 
     def parse_address_candidates(response_body)
       handle_api_errors(response_body)
-
+      puts response_body
```

Comment:
just wanted to flag this in case the linter doesn't, please remove before merging